test/ao/sparsity/test_scheduler.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from torch import nn
+from torch.ao.sparsity import WeightNormSparsifier
+from torch.ao.sparsity import BaseScheduler
+
+from torch.testing._internal.common_utils import TestCase
+
+class ImplementedScheduler(BaseScheduler):
+    def get_sl(self):
+        if self.last_epoch > 0:
+            return [group['sparsity_level'] * 0.5
+                    for group in self.sparsifier.module_groups]
+        else:
+            return list(self.base_sl)
+
+
+class TestBaseScheduler(TestCase):
```

Comment:
Please add tests for verifying (1) Sparsifier is called before scheduler. (2) get_sl method behaves as expected.